### PR TITLE
[libsoundio] Add libsoundio port

### PR DESCRIPTION
**Pull request by UnaNancyOwen**
This pull request will add [libsoundio](https://github.com/andrewrk/libsoundio).

### CODE REVIEW

File: ports/libsoundio/CONTROL [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Source: libsoundio
+Version: 2.0.0
+Description:
```
Comment (Rastaban) [inline ports/libsoundio/CONTROL]: Could you add a description?  There is also an option "Homepage:" tag you can add if the project has one.

**Comment by Rastaban** [comment]
Do you know if the library supports UWP?  If not we can ignore those failures in the CI system.

File: ports/libsoundio/CONTROL (hunk 1 above)
Comment (UnaNancyOwen) [reply ports/libsoundio/CONTROL]: Thanks, I was add them.

**Comment by UnaNancyOwen** [comment]
@Rastaban Probably, This library is not supports UWP.

**Review by Rastaban — APPROVED** [review]

**Comment by UnaNancyOwen** [comment]
@Rastaban I was finished some fixes about libsoundio-config.cmake. It works correctly on Windows and Linux (Ubuntu 18.04 LTS).

**Comment by vicroms** [comment]
LGTM, thanks for the PR!

**Pull request merged**